src/fprime/fbuild/cmake.py

```diff
@@ -280,6 +281,8 @@ def generate_build(
             ),
             args.keys(),
         )
+        # Creating a file to mark the directory as a F Prime directory
+        Path(build_dir / ".fprime-build-dir").touch()
```

Comment:
Looks like there are tests that show `build_dir` won't always be a Path object, so `build_dir / ".fprime-build-dir"` isn't a valid expression (can't use `/` on strings)

Should do instead:

```suggestion
        (Path(build_dir) / ".fprime-build-dir").touch()
```